addons/html_editor/static/src/fields/html_field.js

```diff
@@ -19,17 +20,43 @@ export class HtmlField extends Component {
         useBus(model.bus, "NEED_LOCAL_CHANGES", ({ detail }) =>
             detail.proms.push(this.commitChanges({ shouldInline: true }))
```

Comment:
`commitChanges` takes no args

Reply:
I let it to don t forget it (I need to check if we need it)